src/components/Navbar.js

```diff
@@ -0,0 +1,49 @@
+import { NavLink } from 'react-router-dom';
+import styles from '../styles/Navbar.module.css';
+
+function Navbar() {
+  return (
+    <nav className={styles.nav}>
+      <h1 className={styles.h1}>Math Magicians</h1>
+      <ul className={styles.ul}>
+        <li className={styles.li}>
+          <NavLink
+            to="/"
+            className={({ isActive }) => {
+              const active = isActive ? styles.active : styles.navLink;
+              return active;
+            }}
+          >
+            Home
+          </NavLink>
+        </li>
+        <li>|</li>
+        <li className={styles.li}>
+          <NavLink
+            to="calculator"
+            className={({ isActive }) => {
+              const active = isActive ? styles.active : styles.navLink;
+              return active;
+            }}
+          >
+            Calculator
+          </NavLink>
+        </li>
+        <li>|</li>
+        <li className={styles.li}>
+          <NavLink
+            to="quote"
+            className={({ isActive }) => {
+              const active = isActive ? styles.active : styles.navLink;
+              return active;
+            }}
+          >
+            Quote
+          </NavLink>
+        </li>
+      </ul>
+    </nav>
+  );
+}
+
+export default Navbar;
```

Comment:
- [ ] Please tweak your `Navigation` styling, your navigation is not looking professional and good. Your nav menu should appear on the right-hand side of the screen. But it seems to be centred on position. Following the `wireframe` carefully will help you to be a better developer and also make you a `detail-oriented` person.

### Design Comparison
<table>
<tr>
<th>Provided Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/alex1779/math-magicians/assets/66476329/8ac5f8cf-76f0-4ff0-b2a5-732a4098c979)


</td>
<td>

![image](https://github.com/alex1779/math-magicians/assets/66476329/8b31e2c4-8107-4006-8ce0-ffed6ba3b321)


</td>
</tr>
</table>